incubator/java-microprofile/image/project/Dockerfile

```diff
@@ -23,6 +23,7 @@ RUN cd /project/user-app && mvn -B liberty:install-server
 # Remove quick-start-security.xml since it is only needed during local development.
 COPY ./user-app/src /project/user-app/src
 RUN cd /project/user-app && \
+    echo "QUICK START SECURITY IS NOT SECURE FOR PRODUCTION ENVIRONMENTS. IT IS BEING REMOVED" \
```

Comment:
Should we add this too to java-openliberty ?

Reply:
Ok I'll add it there.